src/styles.css

```diff
@@ -8,7 +8,7 @@ html {
 
 body {
   height: 100%;
-  font-family: Arial;
+  font-family: Arial, sans-serif;
```

Comment:
A generic fallback font should be specified for non-icon fonts. The open-source google font which was previously specified here had a less restrictive licence than Arial. Arial is a commercial font and is not available on Mac or Linux systems. We should definitely have additional non-generic fallbacks for Mac and Linux users.